### ISSUE TITLE
Target project with specified partials using different project's partials

### ISSUE THREAD

**Issue by imolorhe** [assemble/assemble#572]
Hello, I just started using assemble recently and so far it has worked well on a single project basis. However, I am now adding different similar projects to the assemble grunt file and I am having an issue with the partials being used. One of the projects is using partials located in the folder of another project. I have been battling with this for a while now and don't know what else to do. Below is the assemble task in the Gruntfile.js:

```
    assemble: {
        options: {
            flatten: true,
            assets: './',
            layout: 'default.hbs',
            plugins: ['permalinks'],
            layoutdir: 'layouts',
            partials: ['includes/**/*.hbs'],
            data: ['data/**/*.{json,yml}'],
            ext: '.html'
        },
        movies: {
            options: {
                layout: 'movies/default.hbs',
                data: ['data/*.yml'],
                partials: ['includes/movies/*.hbs']
            },
            src: ['templates/movies/*.hbs'],
            dest: './movies/'
        },
        events: {
            options: {
                layout: 'events/default.hbs',
                data: ['data/*.yml'],
                partials: ['includes/events/*.hbs']
            },
            src: ['templates/events/*.hbs'],
            dest: './events/'
        },
        test: {
            options: {
                layout: 'test-layout.hbs'
            },
            src: ['templates/test/*.hbs'],
            dest: 'test/'
        }
    }
```

So basically what is happening is, when I run the events project with `grunt assemble:events`, the project uses partials from `includes/movies/*.hbs` instead of `includes/events/*.hbs`, probably because the partials (like `header.hbs` and `footer.hbs`) have the same name. But it shouldn't still be the case. 

Thanks.
